src/api_fns.py

```diff
@@ -41,23 +41,41 @@ def get_env_info(env:str=ENV, config_file:str='config/config.yaml')->str:
     env_port = config_dict[env]['API port']
     return env_host, env_port
 
-def duplicate(blob_hash: str, cur) -> bool:
+def duplicate(storage_fn_inst: StorageFnSchema, cur) -> dict:
```

Comment:
rename to "check_for_duplicate" or something like that. because you aren't actually duplicating.